bin/tictactoe

```diff
@@ -1,3 +1,21 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+puts "Welcome to Tic-Tac-Toe. How many players will there be?"
+input = gets.chomp
+if input == "0"
+  player_1 = Player::Computer.new("X")
+  player_2 = Player::Computer.new("O")
+elsif input == "1"
+  player_1 = Player::Human.new("X")
+  player_2 = Player::Computer.new("O")
+else
+  player_1 = Player::Human.new("X")
+  player_2 = Player::Human.new("O")
+end
+puts "Player 1 is X, Player 2 is O."
+board = Board.new
+game = Game.new(player_1,player_2,board)
+board.display
+game.play
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

TicTacToeCLI.new.call
```

And then you'd have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.